pkg/connection/manager.go

```diff
@@ -142,7 +143,7 @@ func (m *DetailsManager) FetchConnection(ctx context.Context, so resource.Connec
 }
 
 // PropagateConnection propagate connection details from one resource to another.
-func (m *DetailsManager) PropagateConnection(ctx context.Context, to resource.LocalConnectionSecretOwner, from resource.ConnectionSecretOwner) (propagated bool, err error) { // nolint:interfacer
+func (m *DetailsManager) PropagateConnection(ctx context.Context, to resource.LocalConnectionSecretOwner, from resource.ConnectionSecretOwner) (propagated bool, err error) { //nolint:interfacer // See note below.
```

Comment:
Interfacer has been deprecated, see https://github.com/mvdan/interfacer and golangcilint-list, I'd say it could be even dropped.

Reply:
Hrm, I would expect `nolintlint` to complain if this statement was ineffective.

Reply:
Nono, it's effective as it's still integrated into golangci-lint, but the original project was archived suggesting it's not so useful, which I kind of agree with

Reply:
Nope, removed the nolint directive and did not get a linter warning. I've removed it.